easybuild/easyconfigs/p/ParaView/ParaView-4.4.0-intel-2016a.eb

```diff
@@ -31,6 +32,7 @@ separate_build_dir = True
 configopts = '-DPARAVIEW_INSTALL_DEVELOPMENT_FILES=ON -DVTK_OPENGL_HAS_OSMESA=ON '
 configopts += '-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include -DOPENGL_gl_LIBRARY=$EBROOTMESA/lib/libGL.so '
 configopts += '-DOSMESA_INCLUDE_DIR=$EBROOTMESA/include -DOSMESA_LIBRARY=$EBROOTMESA/lib/libOSMesa.so '
+configopts += '-DVTK_USE_SYSTEM_HDF5=ON '
```

Comment:
@besserox we should make the same change in the other easyconfigs for ParaView 4.4.0 too...